### PR TITLE
Fix world update for Summit-XL steel/Fabtino/Mir100

### PR DESCRIPTION
**Pull request by ad-daniel**
**Description**
World update test fails because of the field order/formatting in the world file.

### CODE REVIEW

**Comment by mael25** [comment]
Thanks! I will do the same for the similar Fabtino world that has just been merged.

**Comment by ad-daniel** [comment]
Looks to be working, check if you can approve it.

**Comment by mael25** [comment]
I think we could take the opportunity of this PR to add  the `renderingDevicePerspectives` in the `.wbproj` to avoid the full screen cameras.

**Comment by mael25** [comment]
I will do the same for the `Mir100`

**Comment by mael25** [comment]
So I took the opportunity of this PR to also:

- [ ] Enable the camera in the demo controllers
- [ ] Choose a proper camera window size for the .wbproj file
- [ ] In the controllers, get the basic timestep from the robot and not redefine it
- [ ] Remove an unnecessary pillar in the Mir100 world and correct the height of its camera

File: projects/objects/factory/conveyors/controllers/keyboard_conveyor_platform/keyboard_conveyor_platform.c [hunk 1]
```diff
@@ -28,13 +28,14 @@
...
 #include <webots/robot.h>
 
-#define TIME_STEP 32
 #define SPEED_INCREMENT 0.1
 #define MAX_SPEED 0.6
 
 int main() {
   wb_robot_init();
 
+  int time_step = wb_robot_get_basic_time_step();
```
Comment (omichel) [inline projects/objects/factory/conveyors/controllers/keyboard_conveyor_platform/keyboard_conveyor_platform.c:37]:
```suggestion
  const int time_step = wb_robot_get_basic_time_step();
```

File: projects/robots/mir/mir100/controllers/keyboard_control/keyboard_control.c [hunk 2]
```diff
@@ -42,6 +42,8 @@
 int main() {
   wb_robot_init();
 
+  int time_step = wb_robot_get_basic_time_step();
```
Comment (omichel) [inline projects/robots/mir/mir100/controllers/keyboard_control/keyboard_control.c:45]:
```suggestion
  const int time_step = wb_robot_get_basic_time_step();
```

File: projects/robots/rec/fabtino/controllers/keyboard_control/keyboard_control.c [hunk 3]
```diff
@@ -39,6 +38,8 @@
 int main() {
   wb_robot_init();
 
+  int time_step = wb_robot_get_basic_time_step();
```
Comment (omichel) [inline projects/robots/rec/fabtino/controllers/keyboard_control/keyboard_control.c:41]:
```suggestion
  const int time_step = wb_robot_get_basic_time_step();
```

File: projects/robots/robotnik/summit_xl_steel/controllers/keyboard_control/keyboard_control.c [hunk 4]
```diff
@@ -40,6 +41,8 @@
 int main() {
   wb_robot_init();
 
+  int time_step = wb_robot_get_basic_time_step();
```
Comment (omichel) [inline projects/robots/robotnik/summit_xl_steel/controllers/keyboard_control/keyboard_control.c:44]:
```suggestion
  const int time_step = wb_robot_get_basic_time_step();
```

**Review by omichel — CHANGES_REQUESTED** [review]
A few const would be nice.

**Comment by ad-daniel** [comment]
Looks good to me, I can't approve it though

**Pull request merged**